daq/docker_test.py

```diff
@@ -48,6 +48,9 @@ def start(self, port, params, callback):
                          "SWITCH_PORT=" + params['switch_port'],
                          "SWITCH_IP=" + params['switch_ip'],
                          "SWITCH_MODEL=" + params['switch_model']]
+        if 'switch_username' in params:   
+            env_vars += ["SWITCH_USERNAME=" + params['switch_username'],
```

Comment:
This is a nice feature. 
I was thinking originally that it could be put inside one of the `module_config.json` when configuring the switch module, something like

```
    "switch": {
      "enabled": true,
      "switch_model": "CISCO_9300",
      "username": "admin",
      "password": "password",
      "poe": {
        "enabled" : true
      }
    },
```

rather than in the `system.conf` file. @grafnu @jhughesbiot what do you think?
If you both think using `system.conf` is better, then we should move the poe boolean inside `system.conf` too. 
It would be best to have all the configuration variables for the test in the same place.